bats/nodejs-examples.bats

```diff
@@ -74,4 +75,15 @@ teardown() {
   tx_template_code=$(graphql_output '.data.txTemplateFindByCode.txTemplate.code')
   [[ "$tx_template_code" != "RECORD_DEPOSIT" ]] || exit 1
 
+  sleep 10
```

Comment:
Please don't use sleep in bats tests - find a way to loop until the expected state is either there or it fails after some timeout.

Reply:
👍 This is actually something I forgot to remove -- missed it in my diff check.